apps/web/src/components/Settings/Dispatcher/ToggleDispatcher.tsx

```diff
@@ -85,6 +88,31 @@ const ToggleDispatcher: FC<Props> = ({ buttonSize = 'md' }) => {
     } catch {}
   };
 
+  const handleIDKitProof = async (result: ISuccessResult) => {
+    const response = await fetch('https://world-id-lens-bridge.vercel.app/api/v1/submit', {
```

Comment:
Are we going to use this endpoint or will it comes under any other subdomain? also I think we can move the URL to constants.